python/doc/examples/graphs/graphs.rst

```diff
@@ -7,4 +7,5 @@ Graphs
     graphs_basics.ipynb
     graphs_fill_area.ipynb
     graphs_loglikelihood_contour.ipynb
+    use-color-palette.ipynb
```

Comment:
I'm sorry, I cant let you add examples until kieran work is merged

Reply:
I understand. However:

1. this PR provides a new feature while Kieran's PR mostly updates the doc (but Kieran's PR also marginally provides new use cases). If the current PR is not merged, it will wait for OT 1.17, what a pity. However, Kieran's PR can be merged afterwards, because it updates the doc, which is mostly online. 
2. Kieran's problems are much larger than this small PR (mostly on continuous integration, because of versions, dependency, etc...). Once Kieran has fixed many other bugs, I guess that porting a small, simple, example will be a blast and will required 15minutes, at worst.

If required, I may remove this example, and add it afterwards? I may also include them within the DrawableImplementation_doc.i.in, which is unchanged by Kieran's PR. 

I wait for your suggestion.

Reply:
yes, drop the example for now

Reply:
Ok, done. I keep it aside to suggest a PR later.

Reply:
also remove its entry in graphs.rst